### ISSUE TITLE
bin/ddms.php MUST account for local autoloader, and possibly Darling Data Management System autoloader

### ISSUE THREAD

**Issue by sevidmusic** [sevidmusic/ddms#17]
After first manual test using as a composer package from context of the Darling Data Management System, it became clear that `bin/ddms.php`'s autoload line fails because the autoloader in use will be the Darling Data Management Systems, so the following from `bin/ddms.php`:

```
<?php

require str_replace('bin', '', __DIR__) . DIRECTORY_SEPARATOR . 'vendor/autoload.php';
...

```

Needs to be refactored to resemble:

```

<?php
if($isInstalledAsComposerPackage) {
    // require  autoloader from Darling Data Management System...
} else { 
    // use local autoloader
    require str_replace('bin', '', __DIR__) . DIRECTORY_SEPARATOR . 'vendor/autoload.php';
}
```

**Comment by sevidmusic** [comment]
This was resolved in commit ece0f9ffa7bc826581e1efa9e5cdc6f803502161 and was merged in c0c886237bedb5365873005affb9cd4d2f1e31aa

**Comment by sevidmusic** [comment]
This was resolved in commit ece0f9ffa7bc826581e1efa9e5cdc6f803502161 and was merged in c0c886237bedb5365873005affb9cd4d2f1e31aa
